cpp/oneapi/dal/table/detail/common.hpp

```diff
@@ -20,78 +20,4 @@
 
```

Comment:
What is the purpose of this file?

Reply:
It's expected to contain common definitions for tables. I assume this will be needed  a bit later, there is a lot of code that can be generalized and reused for tables and can be placed here.